cli/command/formatter/container_test.go

```diff
@@ -471,6 +471,16 @@ func TestDisplayablePorts(t *testing.T) {
 			},
 			"0.0.0.0:0->9988/tcp",
 		},
+		{
+			[]types.Port{
+				{
+					IP:          "::",
+					PrivatePort: 9988,
+					Type:        "tcp",
+				},
+			},
+			expected: "[::]:0->9988/tcp",
```

Comment:
The CI fail with this error:

> #20 59.13 === Errors
> #20 59.13 cli/command/formatter/container_test.go:482:12: mixture of field:value and value elements in struct literal

```suggestion
			"[::]:0->9988/tcp",
```

Reply:
Oh right, I removed the `ports:` but didn't notice the `expected` 🙈